src/Core/AbstractStack.php

```diff
@@ -134,6 +140,30 @@ public function setStackOptions(array $options)
         return $this;
     }
 
+    /**
+     * @since 1.10.0
+     *
+     * @return array
+     */
+    public function getStackVariables()
+    {
+        return $this->stackVariables;
+    }
+
+    /**
+     * @since 1.10.0
+     *
+     * @param array $variables
```

Comment:
Array of what? (I feel like a broken record LOL)

Reply:
fixed